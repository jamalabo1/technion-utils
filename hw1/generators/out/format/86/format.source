D Z 393 1
A Y 149 2
C X 769 3
B X 10 4
B Z 163 5
B X 683 6
B Z 506 7
C Z 224 8
B Y 236 9
C W 722 10
B X 133 11
C X 912 12
B W 835 13
D Y 941 14
C Z 676 15