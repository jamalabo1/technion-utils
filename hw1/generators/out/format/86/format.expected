Blockchain Info:
1
Sender Name: C
Receiver Name: Z
Transaction Value: 676
Transaction timestamp: 15
2
Sender Name: D
Receiver Name: Y
Transaction Value: 941
Transaction timestamp: 14
3
Sender Name: B
Receiver Name: W
Transaction Value: 835
Transaction timestamp: 13
4
Sender Name: C
Receiver Name: X
Transaction Value: 912
Transaction timestamp: 12
5
Sender Name: B
Receiver Name: X
Transaction Value: 133
Transaction timestamp: 11
6
Sender Name: C
Receiver Name: W
Transaction Value: 722
Transaction timestamp: 10
7
Sender Name: B
Receiver Name: Y
Transaction Value: 236
Transaction timestamp: 9
8
Sender Name: C
Receiver Name: Z
Transaction Value: 224
Transaction timestamp: 8
9
Sender Name: B
Receiver Name: Z
Transaction Value: 506
Transaction timestamp: 7
10
Sender Name: B
Receiver Name: X
Transaction Value: 683
Transaction timestamp: 6
11
Sender Name: B
Receiver Name: Z
Transaction Value: 163
Transaction timestamp: 5
12
Sender Name: B
Receiver Name: X
Transaction Value: 10
Transaction timestamp: 4
13
Sender Name: C
Receiver Name: X
Transaction Value: 769
Transaction timestamp: 3
14
Sender Name: A
Receiver Name: Y
Transaction Value: 149
Transaction timestamp: 2
15
Sender Name: D
Receiver Name: Z
Transaction Value: 393
Transaction timestamp: 1